src/client/automation/playback/type/type-text.js

```diff
@@ -145,6 +145,9 @@ function _typeTextToContentEditable (element, text) {
     var endNode             = currentSelection.endPos.node;
     var needProcessInput    = true;
     var needRaiseInputEvent = true;
+    var textInputData       = text;
+
+    text = text === ' ' ? String.fromCharCode(160) : text;
```

Comment:
Text won't be a single char when `typingOptions.paste` is set to `true`. https://devexpress.github.io/testcafe/documentation/test-api/actions/action-options.html#typing-action-options